README.md

```diff
@@ -374,6 +374,86 @@ Defines an interval, in seconds, at which the event log buffer will periodically
 	}
 ```
 
+###event_log_ngx_vars
+
+*Default*: empty
+
+Defines what extra variables from ```ngx.var``` are put to the log event. This is a generic way to extend the alert with extra context. The variable name will be the key of the entry under an ```ngx``` key in the log entry. If the variable is not present as an nginx variable, no item is added to the event.
+
+*Example*:
+
+```lua
+	location / {
+		access_by_lua '
+			-- save the POST data with the alerts
```

Comment:
it doesn't look like the options specified in this example save POST data? copy pasta error?